frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -87,3 +89,20 @@ export const getFlavorDescription = (vm) => {
 
 export const getVMStatusConditions = (vm: VMKind) =>
   _.get(vm, 'status.conditions', []) as VMKind['status']['conditions'];
+
+export const getCDRoms = (vm: VMKind) => getDisks(vm).filter((device) => !!device.cdrom);
+
+export const getContainerImageByDisk = (vm: VMKind, name: string) =>
+  getVolumeContainerImageName(getVolumes(vm).find((vol) => name === vol.name));
+
+export const getURLSourceByDisk = (vm: VMKind, name: string) =>
+  _.get(
+    getDataVolumeTemplates(vm).find((vol) => vol.metadata.name.includes(name)),
+    'spec.source.http.url',
+  );
+
+export const getWindowsToolsURLByDisk = (vm: VMKind, name: string) =>
+  isWindows(vm) ? getURLSourceByDisk(vm, name) : null;
+
+export const getPVCSourceByDisk = (vm: VMKind, cdName: string) =>
```

Comment:
```suggestion
export const getPVCSourceByDisk = (vm: VMKind, diskName: string) =>
```

as it is not specific for cd disks only